### PR TITLE
fix(tools/dev-server): boxen compat + server port

### PR DESCRIPTION
**Pull request by marcoscaceres**

### CODE REVIEW

File: tools/dev-server.js [hunk 1]
```diff
@@ -202,5 +201,6 @@ function printWelcomeMessage(args) {
     borderStyle: "bold",
     backgroundColor: "black",
   };
+  const { default: boxen } = await import("boxen");
```
Comment (sidvishnoi) [inline tools/dev-server.js:204]: Current boxen also has dual module support?

**Review by sidvishnoi — COMMENTED** [review]
Missed server port change?

File: tools/dev-server.js (hunk 1 above)
Comment (marcoscaceres) [reply tools/dev-server.js:204]: No idea... seems to just want this.

**Comment by marcoscaceres** [comment]
> Missed server port change?

heh, yeah, it was in #3848

**Comment by marcoscaceres** [comment]
Merging this so I can keep working on #3848.

**Pull request merged**